### PR TITLE
 code coverage of 40% : Recent Local Storage Items Service

### PR DESCRIPTION
**Pull request by CODEX108**

### CODE REVIEW

File: src/app/core/services/recent-local-storage-items.service.spec.ts [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
   it('should be created', () => {
-    expect(service).toBeTruthy();
+    expect(recentLocalStorageItemsService).toBeTruthy();
+  });
+
+  it('clear(): should delete the specified cache from storage', async () => {
+    const cacheName = 'test-cache';
+    await recentLocalStorageItemsService.clear(cacheName);
+
+    expect(storageService.delete).toHaveBeenCalledWith(cacheName);
```
Comment (jayfyle) [inline src/app/core/services/recent-local-storage-items.service.spec.ts:32]: use `ToHaveBeenCalledOnceWith`

File: src/app/core/services/recent-local-storage-items.service.spec.ts [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
+  });
+
+  it('clearRecentLocalStorageCache(): should clear multiple recent local storage cache', async () => {
+    const clearSpy = spyOn(recentLocalStorageItemsService, 'clear');
+    recentLocalStorageItemsService.clearRecentLocalStorageCache();
+    expect(clearSpy).toHaveBeenCalledWith('advanceProjectCache');
```
Comment (jayfyle) [inline src/app/core/services/recent-local-storage-items.service.spec.ts:38]: Same here

**Review by jayfyle — APPROVED** [review]
Minor changes, rest looks good.

**Review by Dimple16 — APPROVED** [review]
fix the comments and merge

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**